lib/core/file.nit

```diff
@@ -663,6 +663,13 @@ class Path
 		return res
 	end
 
+	# Is `self` the path to a directory ?
```

Comment:
Is the `existing` part necessary ?

If the function returns `true`, the directory necessarily exists, if it does not, it might either exist or not be a directory.
The existence of a file or any other descriptor at this path is the preoccupation of another service.

I can add it nonetheless, but I'm not sure how useful it is to declare it here.